core-services/release-controller/_releases/release-ocp-4.17.json

```diff
@@ -36,12 +36,6 @@
         "name": "periodic-ci-openshift-release-master-nightly-4.17-e2e-agent-ha-dualstack-conformance"
       }
     },
-    "agent-sno-ipv6": {
-      "optional": true,
-      "prowJob": {
-        "name": "periodic-ci-openshift-release-master-nightly-4.17-e2e-agent-sno-ipv6"
-      }
-    },
```

Comment:
Not sure if it's a typo, but I was expecting to see a renaming also here (in the release-ocp-4.x files)

Reply:
yes, those should have been added. I though I added it when I ran `make release-controllers`. Will fix it.